src/main/java/game_entities/Player.java

```diff
@@ -107,8 +132,21 @@ public void addMoney(int add){
      * @param subtract      int representing the amount of money we are trying to subtract
      * @return              return true if the player had enough money to make the payment and false if they do not
      */
+
     public void subtractMoney(int subtract){
-        this.money -= subtract;
+        this.subtractMoney(subtract, null);
+    }
+    public void subtractMoney(int subtract, Player owedPlayer){
+        if(subtract <= this.money) {
+            this.money -= subtract;
+        }
+        else{
+            LiquidateAssetsInterfaceAdapter laia = new LiquidateAssetsInterfaceAdapter(this.turnController, this.mainPanel, this.cardLayout);
```

Comment:
The Player entity should take in ONLY the `turnController` and then pass in to the `turnController.getPlayerOptions()` method the appropriate items.
The `turnController.getPlayerOptions()` method should take the data required for creating a `LiquiditySituation` and create a `LiquiditySituation` which is passed to the `LiquidateAssetsUseCase` (you can change the `TurnController` method to take in those attributes instead of a `LiquiditySituation`,